src/armscan_env/volumes/slicing.py

```diff
@@ -0,0 +1,211 @@
+import logging
+
+import numpy as np
+import SimpleITK as sitk
+from armscan_env.envs.state_action import ManipulatorAction
+
+log = logging.getLogger(__name__)
+
+
+def padding(original_array: np.ndarray) -> np.ndarray:
+    """Pad an array to make it square.
+
+    :param original_array: array to pad
+    :return: padded array.
+    """
+    # Find the maximum dimension
+    max_dim = max(original_array.shape)
+
+    # Calculate padding for each dimension (left and right)
+    padding_x_left = (max_dim - original_array.shape[0]) // 2
+    padding_x_right = max_dim - original_array.shape[0] - padding_x_left
+
+    padding_y_left = (max_dim - original_array.shape[1]) // 2
+    padding_y_right = max_dim - original_array.shape[1] - padding_y_left
+
+    padding_z_left = (max_dim - original_array.shape[2]) // 2
+    padding_z_right = max_dim - original_array.shape[2] - padding_z_left
+
+    # Pad the array with zeros
+    padded_array = np.pad(
+        original_array,
+        (
+            (padding_x_left, padding_x_right),
+            (padding_y_left, padding_y_right),
+            (padding_z_left, padding_z_right),
+        ),
+        mode="constant",
+    )
+
+    # Verify the shapes
+    log.debug(
+        f"Original Array Shape: {original_array.shape}\n"
+        f"Padded Array Shape: {padded_array.shape}",
+    )
+
+    return padded_array
+
+
+class EulerTransform:
```

Comment:
This should be in the same module as ManipulatorAction, it's not related to slicing